doc/developers-guide/domain_management.md

```diff
@@ -101,6 +101,21 @@ Configure host-type first to delete such domains.
 
 ## Service options
 
+```toml
+[services.service_domain_db]
+  pool_name = "global"
+  event_cleaning_interval = 1800
+  event_max_age = 7200
+```
+
+### `pool_name`
+
+Pool name to use for a database connection.
+
+* **Syntax:** atom
+* **Default:** `global`
+* **Example:** `pool_name = "global"`
+
```

Comment:
The problem with this is that we don't have such thing as a "pool name". In the code it has the type `host_type_or_global()`, so I would suggest something like the following:

```suggestion
### `db_pool`

By default, this service uses the RDBMS connection pool configured with the scope `"global"`.
You can put a specific host type there to use the pool with the `"host"` or `"single_host"`scope for that particular host type. See [PUT A LINK HERE] for more information about the pool scopes.

* **Syntax:** string
* **Default:** `global`
* **Example:** `db_pool = "my_host_type"`
```

The goal here is to make the user aware of what this name means and how to set up the corresponding pool.